tests/cli/test_cli.py

```diff
@@ -1890,22 +1851,70 @@ def test_search(ctx):
         check_equals(uuid1, _run_command([cl, 'search', 'uuid=' + uuid1[0:8] + '.*', '-u']))
         check_equals(uuid1, _run_command([cl, 'search', 'uuid=' + uuid1[0:8] + '%', '-u']))
         check_equals(uuid1, _run_command([cl, 'search', 'uuid=' + uuid1, 'name=' + name, '-u']))
+        check_equals(
+            uuid1 + '\n' + uuid2, _run_command([cl, 'search', 'name=' + name, 'id=.sort', '-u'])
+        )
+        check_equals(
+            uuid1 + '\n' + uuid2,
+            _run_command([cl, 'search', 'uuid=' + uuid1 + ',' + uuid2, 'id=.sort', '-u']),
+        )
+        check_equals(
+            uuid2 + '\n' + uuid1, _run_command([cl, 'search', 'name=' + name, 'id=.sort-', '-u'])
+        )
+        check_equals('2', _run_command([cl, 'search', 'name=' + name, '.count']))
+        size1 = float(_run_command([cl, 'info', '-f', 'data_size', uuid1]))
+        size2 = float(_run_command([cl, 'info', '-f', 'data_size', uuid2]))
+        check_equals(
+            size1 + size2, float(_run_command([cl, 'search', 'name=' + name, 'data_size=.sum']))
+        )
+
+        """Search for floating bundles"""
```

Comment:
nit: I think we use comment style `#` rather than docstrings for mid-method comments?